service/server.py

```diff
@@ -25,6 +25,8 @@
 POSTCODE_REGEX = re.compile(address_utils.BASIC_POSTCODE_REGEX)
 NOF_SECS_BETWEEN_LOGINS = 1
 LOGGER = logging.getLogger(__name__)
+INCLUDE_TIME_IN_DATE_FORMAT = True
+IGNORE_TIME_IN_DATE_FORMAT = False
```

Comment:
I see now. Probably the best way to achieve this goal would be by using named parameter, e.g.:

    _create_string_date(date_to_format, needs_time=false):